internal/host/hostutil/update_host.go

```diff
@@ -68,12 +68,15 @@ func UpdateHostStatus(ctx context.Context, log logrus.FieldLogger, db *gorm.DB,
 			hostId, infraEnvId, srcStatus, newStatus)
 	}
 
+	if host, err = common.GetHostFromDB(db, infraEnvId.String(), hostId.String()); err != nil {
```

Comment:
why do you need to get the host when you already get it from line 65?

Reply:
you're right. indeed what we get from `UpdateHost()` is enough. I removed it.